spring-cloud-gcp-vision/pom.xml

```diff
@@ -24,5 +24,18 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-core</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-storage</artifactId>
+			<optional>true</optional>
```

Comment:
Is it optional? VisionTemplate depends on Storage in all cases.

Reply:
Just for OCR, I believe.

Reply:
Yeah I wasn't 100% sure to be honest; it's what I had before but Mike suggested it could be marked optional since only the OCR template uses storage but not the general one. I figure once I merge this with my sample I'll be able to figure it out.